src/frontend/barnetilsyn/tekster/opphold.ts

```diff
@@ -90,3 +122,144 @@ export const mottarPengestøtteInnhold: MottarDuPengestøtteInnhold = {
     },
 };
+
+const årsakAlternativer: Record<ÅrsakOppholdUtenforNorge, TekstElement<string>> = {
+    JOBB: {
+        nb: 'Jobbet',
+    },
+    STUDIER: {
+        nb: 'Studerte',
+    },
+    MEDISINSK_BEHANDLING: {
+        nb: 'Fikk medisinsk behandling',
+    },
+    FERIE: {
+        nb: 'Var på ferie',
+    },
+    FAMILIE_BESØK: {
+        nb: 'Besøkte familie',
+    },
+    ANNET: {
+        nb: 'Annet',
+    },
+};
+const siste12mndInnhold: OppholdInnhold = {
+    select_hvilket_land: {
+        nb: 'Hvilket land har du oppholdt deg i?',
+    },
+    feilmelding_hvilket_land: {
+        nb: 'Du må velge hvilket land du har oppholdt deg i.',
+    },
+
+    checkbox_årsak: {
+        header: {
+            nb: 'Hva gjorde du i dette landet?',
+        },
+        alternativer: årsakAlternativer,
+    },
+    feilmelding_årsak: {
+        nb: 'Du må velge hva du gjorde i landet du oppholdt deg i.',
+    },
+    dato: {
+        label: {
+            nb: 'Når var du i dette landet?',
+        },
+        fom: {
+            nb: 'Fra',
+        },
+        tom: {
+            nb: 'Til',
+        },
+        feilmelding_fom: {
+            nb: 'Du må velge fra-dato.',
+        },
+        feilmelding_tom: {
+            nb: 'Du må velge til-dato.',
+        },
+        feilmelding_tom_før_fom: {
+            nb: 'Til-dato må være eller etter fra-dato.',
+        },
+    },
+};
+const neste12mndInnhold: OppholdInnhold = {
+    select_hvilket_land: {
+        nb: 'Hvilket land skal du oppholde deg i?',
+    },
+    feilmelding_hvilket_land: {
+        nb: 'Du må velge hvilket land du skal oppholde deg i.',
+    },
+
+    checkbox_årsak: {
+        header: {
+            nb: 'Hva skal du gjøre i dette landet?',
+        },
+        alternativer: årsakAlternativer,
+    },
+    feilmelding_årsak: {
+        nb: 'Du må velge hva du skal gjøre i landet du skal oppholde deg i.',
+    },
+    dato: {
+        label: {
+            nb: 'Når skal du være i dette landet?',
+        },
+        fom: {
+            nb: 'Fra',
+        },
+        tom: {
+            nb: 'Til',
+        },
+        feilmelding_fom: {
+            nb: 'Du må velge fra-dato.',
+        },
+        feilmelding_tom: {
+            nb: 'Du må velge til-dato.',
+        },
+        feilmelding_tom_før_fom: {
+            nb: 'Til-dato må være eller etter fra-dato.',
+        },
+    },
+};
+export const oppholdUtenforNorgeInnhold: OppholdUtenforNorgeInnhold = {
+    dineOpphold: {
+        nb: 'Dine opphold utenfor Norge siste 12 mnd:',
+    },
+    radioSiste12mnd: {
+        header: {
+            nb: 'Har du oppholdt deg utenfor Norge i løpet av de siste 12 månedene?',
+        },
+        beskrivelse: {
+            nb: 'Opphold under 5 uker trenger du ikke opplyse om.',
+        },
+        alternativer: jaNeiAlternativer,
+    },
+    feilmelding_radioSiste12mnd: {
+        nb: 'Du må svare på om du oppholdt deg utenfor Norge.',
```

Comment:
```suggestion
        nb: 'Du må svare på om du har oppholdt deg utenfor Norge.',
```
se [kommentar om perfektum](https://github.com/navikt/tilleggsstonader-soknad/pull/269/files#r1549673277)

Reply:
[460c5de](https://github.com/navikt/tilleggsstonader-soknad/pull/269/commits/460c5de997a301dc1ce48d54b4ff11907c5af61c)